### PR TITLE
convert FIP values to LAB scale

### PR DESCRIPTION
**Pull request by totto82**
Add unit conversion of FIP output in case of LAB unit system.

### CODE REVIEW

**Comment by totto82** [comment]
jenkins build this please

**Comment by andlaus** [comment]
Thanks for digging this up. It's kind of annoying that something like this needs to be done manually on such a high level of abstraction (yes, this is -- in my opinion -- another case of my favorite kind of issue: a layering violation), but the patch is fine and it does regress the status quo. I'll merge once jenkins is happy.

File: ebos/ecloutputblackoilmodule.hh [hunk 1]
```diff
@@ -1256,6 +1256,17 @@ private:
...
         }
+        else if (units.getType() == Opm::UnitSystem::UnitType::UNIT_TYPE_LAB) {
+            Scalar scc = Opm::unit::cubic(Opm::prefix::centi * Opm::unit::meter); //standard cubic cm.
+            fip[FipDataType::WaterInPlace] = Opm::unit::convert::to(fip[FipDataType::WaterInPlace], scc);
+            fip[FipDataType::OilInPlace] = Opm::unit::convert::to(fip[FipDataType::OilInPlace], scc);
+            fip[FipDataType::OilInPlaceInLiquidPhase] = Opm::unit::convert::to(fip[FipDataType::OilInPlaceInLiquidPhase], scc);
+            fip[FipDataType::OilInPlaceInGasPhase] = Opm::unit::convert::to(fip[FipDataType::OilInPlaceInGasPhase], scc);
+            fip[FipDataType::GasInPlace] = Opm::unit::convert::to(fip[FipDataType::GasInPlace], scc);
+            fip[FipDataType::GasInPlaceInLiquidPhase] = Opm::unit::convert::to(fip[FipDataType::GasInPlaceInLiquidPhase], scc);
+            fip[FipDataType::GasInPlaceInGasPhase] = Opm::unit::convert::to(fip[FipDataType::GasInPlaceInGasPhase], scc);
```
Comment (joakim-hove) [inline ebos/ecloutputblackoilmodule.hh]: Why is this needed - the output layer converts from SI to whatever?

Comment (totto82) [reply ebos/ecloutputblackoilmodule.hh]: It is used for the .PRT .DBG output.

Comment (joakim-hove) [reply ebos/ecloutputblackoilmodule.hh]: OK - I guess the "proper" fix is to do the unit conversion in the PRT output routine, but that is of course a bigger change. I am not familiar with the PRT output.
